app/models/user.rb

```diff
@@ -17,13 +17,19 @@ class User < ApplicationRecord
 
   pg_search_scope :search, against: %i(name email about location level_of_availability)
 
-  def volunteered_for_project? project
+  def volunteered_for_project?(project)
     self.volunteered_projects.where(id: project.id).exists?
   end
 
   def has_complete_profile?
     self.about.present? && self.profile_links.present? && self.location.present?
   end
+  
+  def has_correct_skills?(project)
+    project_skills = project.skills.map(&:name)
+    return true if project_skills.include?("Anything")
```

Comment:
Can you install rubocop? Have to improve it a bit, but for ruby we prefer single quotes.